lib/flutter_redux.dart

```diff
@@ -428,22 +424,24 @@ class _StoreStreamListener<S, ViewModel> extends StatefulWidget {
 
 class _StoreStreamListenerState<S, ViewModel>
     extends State<_StoreStreamListener<S, ViewModel>> {
-  Stream<ViewModel> stream;
-  ViewModel latestValue;
+  Stream<ViewModel> _stream;
+  ViewModel _latestValue;
+  Object _latestError;
```

Comment:
Is there a reason `_latestError` isn't a ConverterError? It seems from this patch that that is the only type that's ever assigned to this property.